src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -209,8 +209,8 @@ function radiation_model_cache(
             center_temperature = NaN, # initialized in callback
             surface_temperature = NaN, # initialized in callback
             surface_emissivity = 1,
-            direct_sw_surface_albedo = 0.38,
-            diffuse_sw_surface_albedo = 0.38,
+            direct_sw_surface_albedo = NaN, # initialized in callback
+            diffuse_sw_surface_albedo = NaN, # initialized in callback
```

Comment:
Note that `NaN` is Float64, so maybe we want to be more careful with types here.

Reply:
I've followed how we initialize the rest of the cache here, but that is a great point, and we should look into it. I opened an issue here: https://github.com/CliMA/ClimaAtmos.jl/issues/2695. This is maybe something that could be relevant to the float type instabilities @sriharshakandala , @juliasloan25  and @akshaysridhar  have been dealing with. 🤔